templates/stamp/mongodb-ubuntu-install.sh

```diff
@@ -346,4 +346,4 @@ configure_db_users
 configure_replicaset
```

Comment:
christy will add conditions to install_mongodb and start_mongodb based on the ubuntu os version. i reverted my similar changes to reduce conflicts